src/main/java/com/celements/web/plugin/cmd/AttachmentURLCommand.java

```diff
@@ -21,6 +21,7 @@
 
 import static com.celements.common.MoreOptional.*;
 import static com.celements.execution.XWikiExecutionProp.*;
+import static org.python.google.common.base.Strings.*;
```

Comment:
Do you really need "python" Strings not `com.google.common.base.Strings` instead?